ci-operator/step-registry/baremetal/lab/agent/install/network/static/baremetal-lab-agent-install-network-static-workflow.yaml

```diff
@@ -7,6 +7,8 @@ workflow:
       - chain: baremetal-lab-agent-install
     post:
       - chain: baremetal-lab-post
+    env:
+      CLUSTER_WIDE_PROXY: "true"
```

Comment:
We don't want CLUSTER_WIDE_PROXY true by default for ipv4.

Reply:
This is for testing purpose.